libopae/pluginmgr.c

```diff
@@ -221,6 +222,7 @@ STATIC void opae_plugin_mgr_detect_platform(uint16_t vendor, uint16_t device)
 
 		if (platform_data_table[i].vendor_id == vendor &&
 		    platform_data_table[i].device_id == device) {
+			platforms_detected++;
```

Comment:
The rationale is that we would be able to try platform detection again if we failed to detect anything the first time?

Reply:
Yes